### PR TITLE
dashboards: remove apiserver_request_total client label

### PR DESCRIPTION
**Pull request by hypnoglow**
Related issue: https://github.com/kubernetes/perf-tests/issues/1002

### CODE REVIEW

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def api_call_latency(title, verb, scope, threshold):
 CLUSTERLOADER_PANELS = [
```
Comment (mm4tt) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Actually, let's get rid of this whole panel. Without being able to plot requests coming from clusterloader there is no point in having it.

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 2]
```diff
@@ -72,12 +72,12 @@ def api_call_latency(title, verb, scope, threshold):
     ),
     d.simple_graph(
         "Pod creations",
-        'sum(irate(apiserver_request_total{verb="POST", resource="pods", subresource=""}[1m])) by (client)',
```
Comment (mm4tt) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Let's not get rid of the sum aggregation. We want to see an overall pod creation rate in the cluster, without sum you'll get a separate graph for every unique tuple of prometheus labels, e.g. we can have multiple api-servers with different labels, etc.

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 3]
```diff
@@ -72,12 +72,12 @@ def api_call_latency(title, verb, scope, threshold):
...
+        'irate(apiserver_request_total{verb="POST", resource="pods", subresource=""}[1m])',
         yAxes=g.single_y_axis(format=g.OPS_FORMAT),
     ),
     d.simple_graph(
         "Pod bindings",
-        'sum(irate(apiserver_request_total{verb="POST", resource="pods", subresource="binding"}[1m])) by (client)',
```
Comment (mm4tt) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Same here

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py (hunk 1 above)
Comment (hypnoglow) [reply clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Removed and `CLUSTERLOADER_PANELS ` and `api_call_latency` definition that is not used anymore.

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py (hunk 2 above)
Comment (hypnoglow) [reply clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Fixed, now only `by (client)` is dropped.

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 4]
```diff
@@ -18,52 +18,6 @@
...
-                % {"verb": verb, "scope": scope},
-                legendFormat="{{verb}} {{scope}}/{{resource}}",
-            ),
-        ],
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
-    )
-
-
-CLUSTERLOADER_PANELS = [
-    d.simple_graph(
```
Comment (mm4tt) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Sorry, my bad. I meant to delete only this graph (requests), the other graphs (api-call-latency graphs) are actually useful and don't depend on the client label.

Comment (hypnoglow) [reply clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: No problem, `CLUSTERLOADER_PANELS` is back with only first graph deleted.

**Comment by mm4tt** [comment]
/lgtm
/approve

Thanks, @hypnoglow !

**Comment by wojtek-t** [comment]
Closed and reopened to trigger travis.

**Pull request merged**